src/components/common/ColonyActions/ActionsListItem/ActionsListItemMeta.tsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import { defineMessages, FormattedMessage } from 'react-intl';
+
+import Icon from '~shared/Icon';
+import Numeral from '~shared/Numeral';
+import { useColonyContext } from '~hooks';
+import { findDomain } from '~utils/domains';
+import { formatText } from '~utils/intl';
+import { Domain } from '~types';
+
+import styles from './ActionsListItemMeta.css';
+
+const displayName = 'common.ColonyActions.ActionsListItemMeta';
+
+const MSG = defineMessages({
+  domain: {
+    id: `${displayName}.domain`,
+    defaultMessage: 'Team {domainId}',
+  },
+  titleCommentCount: {
+    id: `${displayName}.titleCommentCount`,
+    defaultMessage: `{formattedCommentCount} {commentCount, plural,
+        one {comment}
+        other {comments}
+      }`,
+  },
```

Comment:
We won't have comments enabled for the initial launch.

So either remove these, or just comment them out